src/modules/market/containers/market-outcome-charts.js

```diff
@@ -34,8 +35,8 @@ const mapStateToProps = (state, ownProps) => {
   return {
     currentBlock: state.blockchain.currentBlockNumber || 0,
     currentTimestamp: selectCurrentTimestamp(state),
-    minPrice: market.minPrice || 0,
-    maxPrice: market.maxPrice || 0,
+    minPrice: market.minPrice || createBigNumber(0),
```

Comment:
@pgebheim Where does `market.minPrice` become a BigNumber?

Reply:
In the market selector.

Reply:
In the market selector.